Index: src/api/index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/api/index.js	(revision 3a9037b42f7e9d2cd3df5ea1709789e6c46b0b0b)
+++ src/api/index.js	(date 1594621128268)
@@ -1,0 +1,14 @@
+import * as axios from 'axios'
+/*eslint-disable*/
+
+const instance = axios.create({
+    baseURL: 'https://v-rap.herokuapp.com/'
+})
+
+export const RegisterApi = (regData) => {
+    return instance.post(`api/auth/register`, regData)
+}
+
+export const LoginApi = (loginData) => {
+    return instance.post(`api/auth/login`, loginData)
+}
